lambdas/db-migration/package.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -e
+
+export PATH="../../node_modules/.bin:${PATH}"
+
+rm -rf dist
+
+tsc
+
+cp package.json dist/lambda
+
+(
+  cd dist/lambda
+
+  npm install --production
+
+  rm -f package.json package-lock.json
+
+  zip -r ../lambda.zip migrations node_modules index.js
```

Comment:
Does this create dist/lambda/lambda.zip? Because TF seems to be looking for just dist/lambda.zip

Reply:
This creates `dist/lambda.zip`